File: 1-two-sum/1-two-sum.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        
        vector<int> out(2,0);
        
        for(int i = 0 ; i < nums.size() ; i++){
            
            for(int j=i+1 ; j < nums.size() ; j++){
                
                if(nums.at(i) + nums.at(j) == target){
                    
                    out.at(0) = i;
                    out.at(1) = j;
                    
                    
                    return out;
                }
                
            }
            
        }
        return out;
    }
};

File: 1-two-sum/1-two-sum.js

var twoSum = function(nums, target) {
    let n = nums.length;
    for(let i=0;i<n;i++){
        for(let j=i+1;j<n;j++){
            if(nums[i]+nums[j]===target){
                return[i,j];
            }
        }
    }
    return[0,0];
};